### ISSUE TITLE
Each time I switch projects, Spanish dictionary stops working.

### ISSUE THREAD

**Issue by zomars** [streetsidesoftware/vscode-cspell-dict-extensions#46]
To fix it I have to:

1. Open JSON Settings and remove `es` from `"cSpell.language": "es,en"`
2. Save
3. Re-add the `es` option.
4. Save again.

And it starts working again.

**Comment by brunowego** [comment]
@zomars this approach works for me. 👍

**Comment by zomars** [comment]
My fix stoped working for me, now I have to do the following:

1. Open VS Code command palette (`cmd + shift + p` on mac)
1. Type `Show Spell Checker Configuration Info` and press enter
1. Go to User tab:
![image](https://user-images.githubusercontent.com/3504472/98277035-17168d80-1f54-11eb-8867-37bbf565a3a0.png)
1. Open VS Code command palette again
1. Type `Enable Spanish Spell Checker Dictionary` and press enter

This error appears:

![image](https://user-images.githubusercontent.com/3504472/98277250-61980a00-1f54-11eb-908d-da0a30c08622.png)


```
Command 'Enable Spanish Spell Checker Dictionary' resulted in an error (command 'cSpell.enableSpanish' not found)
```

**Comment by zomars** [comment]
The Extensions Host log shows this:

```
[2020-11-05 07:42:53.974] [exthost] [info] ExtensionService#loadCommonJSModule file:///Users/zomars/.vscode/extensions/streetsidesoftware.code-spell-checker-1.9.2/dist/extension
[2020-11-05 07:42:53.974] [exthost] [info] ExtensionService#_doActivateExtension streetsidesoftware.code-spell-checker {"startup":true,"extensionId":{"value":"streetsidesoftware.code-spell-checker-spanish","_lower":"streetsidesoftware.code-spell-checker-spanish"},"activationEvent":"*"}
[2020-11-05 07:42:53.974] [exthost] [info] ExtensionService#loadCommonJSModule file:///Users/zomars/.vscode/extensions/streetsidesoftware.code-spell-checker-1.9.2/dist/extension
```

Note that the path is missing the `.js` file extension:

![image](https://user-images.githubusercontent.com/3504472/98277797-ff8bd480-1f54-11eb-9242-6de3697a1ac7.png)

**Comment by Jason3S** [comment]
@zomars Thank you for looking into this.
I wonder if the behavior of VS Code change. The `.js` extension should not be needed.

**Comment by Jason3S** [comment]
The spell checker seem to be loading just fine. It is the Spanish dictionary extension that seems to be stuck on load. The error you showed:

![image](https://user-images.githubusercontent.com/3504472/98277250-61980a00-1f54-11eb-908d-da0a30c08622.png)

Means the Spanish dictionary extension only partially loaded when VS Code installed it. Try uninstalling it in VS Code. You might need to delete it from `~/.vscode/extenstion/streetsidesoftware.code-spell-checker-spanish*`

![image](https://user-images.githubusercontent.com/3740137/98454292-641c7e80-2163-11eb-9a9a-b42d2dfa5bf1.png)

**Comment by Jason3S** [comment]
You should see the extension load in the extension host:
![image](https://user-images.githubusercontent.com/3740137/98454327-b8bff980-2163-11eb-9641-7613c653574d.png)

**Comment by Jason3S** [comment]
@zomars I have updated all the language extensions so they load faster. I think that was the reason it would load some times and not others. If an extension does not load fast enough, it is killed.

**Comment by Jason3S** [comment]
I believe this is fixed by #56
